conf/schemas/cloudtrail.json

```diff
@@ -22,26 +22,31 @@
   "cloudtrail:events": {
     "schema": {
       "additionalEventData": {},
+      "addendum": {},
       "apiVersion": "string",
       "awsRegion": "string",
       "errorCode": "string",
       "errorMessage": "string",
+      "eventCategory": "string",
       "eventID": "string",
       "eventName": "string",
       "eventSource": "string",
       "eventTime": "string",
       "eventType": "string",
       "eventVersion": "string",
+      "insightDetails": {},
       "managementEvent": "boolean",
       "readOnly": "boolean",
       "recipientAccountId": "string",
       "requestID": "string",
       "requestParameters": {},
       "resources": [],
       "responseElements": {},
+      "sessionCredentialFromConsole": "boolean",
```

Comment:
```
sessionCredentialFromConsole
Shows whether or not an event originated from a AWS Management Console session. This field is not shown unless the value is true, meaning that the client that was used to make the API call was either a proxy or an external client. If a proxy client was used, the tlsDetails event field is not shown.

Since: 1.08

Optional: True
```